packages/react/__tests__/AdvancedVideo.test.tsx

```diff
@@ -80,6 +81,24 @@ describe('AdvancedVideo', () => {
     }, 0);// one tick
   });
 
+  it('should contain poster when "auto" is passed as cldPoster', function (done) {
+    const component = mount(<AdvancedVideo cldVid={cloudinaryVideo} cldPoster="auto" />);
+
+    setTimeout(() => {
+      expect(component.html()).toContain('poster="https://res.cloudinary.com/demo/video/upload/q_auto/f_auto:image/sample"');
```

Comment:
what is `f_auto:image`?

Reply:
I wrote a test for this "auto" behavior starting with a failing assertion. What I got back was this URL so I used it in the test. 
I did not think much about if it's correct or not - it works, so it's correct. 

I agree it's strange but it's probably the most important _part_ here that allows getting an image from the video, right? Maybe there is some other and betrer way?

Reply:
In our other libraries we use `jpg` format for poster. 
Also `f_auto:image` is not documented in our documentation, it might be sitting under some feature flag and might not work for all customers. The fact that it works can be a side effect and might break/change at any moment.

Reply:
I've also never seen `f_auto:image` before, learn every day 😮 
if it's standard and not under a feature flag, it might be a better alternative since it'll bring the best `image`, and not just force a jpg.

but that does require verification.

Reply:
But let's keep in mind one thing - this is only in the test, the actual URL is generated from SDK so I don't think we should really focus much on it. If it changes it will change via SDK and only test will start failing one day.